### PR TITLE
home and not found pages are complete

### PR DESCRIPTION
**Pull request by Mamba240824**

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
     <>
-
+      <div className="welcome-message">
+        <h1>Welcome to The Trap!</h1>
+        <p>Welcome to the TrapHouse, where all of your home and house needs can be met. Do you want a beautifully spacious kitchen to enhance your cooking skills? A large living room with bay windows for you to assemble with your partners and nem? Want to have enough rooms and couches for the entire family to fall asleep from the itis after a cookout? Look no further. We here at the TrapHouse will find you the perfect place to settle down and live out all your hood-rich dreams.
+          <div></div>
+        <NavLink to="/listings" className="explore-button">Explore Listings</NavLink>
+        </p>
+        <div className="backgroundimage" > <img src={HomeBackgroundImage}
+        alt="..."
```
Comment (sjproctor) [inline src/pages/Home.js]: All images should have alt tags that describe the image.

File: src/pages/Home.js [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+        style={{ width: "100%", height: "auto" }} /> </div>
```
Comment (sjproctor) [inline src/pages/Home.js:15]: I would suggest moving your styling to your css

File: src/pages/Home.js [hunk 3]
```diff
@@ -1,10 +1,23 @@
...
 
-const Home = () => {
+const Home = ({ currentUser }) => {
   return (
     <>
-
+      <div className="welcome-message">
+        <h1>Welcome to The Trap!</h1>
+        <p>Welcome to the TrapHouse, where all of your home and house needs can be met. Do you want a beautifully spacious kitchen to enhance your cooking skills? A large living room with bay windows for you to assemble with your partners and nem? Want to have enough rooms and couches for the entire family to fall asleep from the itis after a cookout? Look no further. We here at the TrapHouse will find you the perfect place to settle down and live out all your hood-rich dreams.
+          <div></div>
```
Comment (sjproctor) [inline src/pages/Home.js]: Looks like a random div snuck in here

File: src/__tests__/Home.test.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { render, screen } from "@testing-library/react"
+import Home from "../pages/Home"
+import { BrowserRouter } from "react-router-dom"
+
+describe("<Home />", () => {
+    it("renders without crashing", () => {
+        const div = document.createElement("div")
+    render(<Home />, div)
```
Comment (sjproctor) [inline src/__tests__/Home.test.js]: Can I get this line 8 tabbed over to be inline with everything else please!

File: src/__tests__/NotFound.test.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+import NotFound from "../pages/NotFound"
+
+describe("<NotFound />", () => {
+    it("renders without crashing", () => {
+        const div = document.createElement("div")
+    render(<NotFound />, div)
+        const notFoundImage = screen.getByRole("img")
+            expect(notFoundImage).toHaveAttribute("src", "NotFoundImage.gif")
+            expect(notFoundImage).toHaveAttribute("alt", "NotFound.gif")
+    })
```
Comment (sjproctor) [inline src/__tests__/NotFound.test.js:11]: This it block needs a little indentation love too.

**Review by sjproctor — COMMENTED** [review]
Few minor things to fix.

File: src/pages/Home.js (hunk 1 above)
Comment (Sakaa4Short) [reply src/pages/Home.js]: Corrected the alt tag with description.

File: src/pages/Home.js (hunk 3 above)
Comment (Sakaa4Short) [reply src/pages/Home.js]: Extra div added to create a line break for styling. <br> didn't work but the div tag was removed and another solution will be implemented on the next upload.

File: src/__tests__/NotFound.test.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import { render, screen } from "@testing-library/react"
+import NotFound from "../pages/NotFound"
+
+describe("<NotFound />", () => {
+    it("renders without crashing", () => {
+        const div = document.createElement("div")
+        render(<NotFound />, div)
+        const notFoundImage = screen.getByRole("img")
+            expect(notFoundImage).toHaveAttribute("src", "NotFoundImage.gif")
+            expect(notFoundImage).toHaveAttribute("alt", "NotFound.gif")
```
Comment (elysemontano) [inline src/__tests__/NotFound.test.js]: These are indented a little too far over.

File: src/pages/NotFound.js [hunk 7]
```diff
@@ -1,9 +1,19 @@
...
     <>
-
+     <div className="NF">
+      <img src={RedBlue} alt="five-Ohhh" className="NotFound-gif" />
+    </div>
+    <div className="not-found-container">
+      <h1>404</h1>
+      <h3>I dont know what you were looking for but you got found!</h3>
+      <NavLink to="/" className="home-button">Return Home</NavLink>
+    </div>
```
Comment (elysemontano) [inline src/pages/NotFound.js]: Check indentation

**Review by elysemontano — CHANGES_REQUESTED** [review]
Left some additional comments about indentation

**Review by elysemontano — APPROVED** [review]

**Pull request merged**